src/styles/blocks/header.scss

```diff
@@ -9,5 +9,13 @@
     padding: 10px;
     width: 40px;
     height: 40px;
+
+    transition: all 0.3s cubic-bezier(0.25, 0.1, 0.25, 1);
```

Comment:
Don't use transition all. Include properties that you want to use transition for.